### ISSUE TITLE
PDF button produces an error

### ISSUE THREAD

**Issue by ebruchez** [orbeon/orbeon-forms#4739]
- Liferay 7.2.1 CE GA2
- Controls form

**Comment by ebruchez** [comment]
Error on the portlet side:

```
javax.portlet.PortletException: java.lang.IllegalArgumentException: application/pdf is an unsupported content type
	at org.orbeon.oxf.portlet.OrbeonProxyPortlet.$anonfun$doView$2(OrbeonProxyPortlet.scala:130)
	at org.orbeon.oxf.portlet.OrbeonProxyPortlet$.withRootException(OrbeonProxyPortlet.scala:426)
	at org.orbeon.oxf.portlet.OrbeonProxyPortlet.$anonfun$doView$1(OrbeonProxyPortlet.scala:130)
	at org.orbeon.oxf.portlet.OrbeonProxyPortlet.$anonfun$doView$1$adapted(OrbeonProxyPortlet.scala:129)
```

Obviously it's wrong to load the PDF using `doView`.

**Comment by ebruchez** [comment]
From the client, with 2020.1:

```
http://localhost:8080/web/guest/home?p_p_id=orbeonformsproxyportlet_WAR_proxyportlet_INSTANCE_WPIqsj7TJ1dr&
p_p_lifecycle=1&
p_p_state=normal&
p_p_mode=view&
_orbeonformsproxyportlet_WAR_proxyportlet_INSTANCE_WPIqsj7TJ1dr_orbeon.path=%2Fxforms-server-submit&
p_auth=nL1sVxC3
```

With 2019.1:

```
http://localhost:8080/web/guest/home?p_p_id=orbeonformsproxyportlet_WAR_proxyportlet_INSTANCE_WPIqsj7TJ1dr&
p_p_lifecycle=2&
p_p_state=normal&
p_p_mode=view&
p_p_resource_id=%2Fxforms-server-submit&
p_p_cacheability=cacheLevelPage
```

So we produce, in the response, an action URL, not a resource URL. This explains the problem.

**Comment by ebruchez** [comment]
Probably regressed with #4385/#4386.

**Comment by ebruchez** [comment]
Verified.

**Comment by ebruchez** [comment]
This was fixed with 2020.1.
